<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-RN9H6QU" failures="0" tests="11" name="com.api.auto.testcase.TC_API_CreateWork2" time="1.226" errors="1" timestamp="2023-07-05T17:56:33 ICT" skipped="10">
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC01_Validate201Created" time="0.000">
    <skipped/>
  </testcase> <!-- TC01_Validate201Created -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC01_Validate200Ok" time="0.000">
    <skipped/>
  </testcase> <!-- TC01_Validate200Ok -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="init" time="1.226">
    <error message="Header value cannot be null" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Header value cannot be null
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:60)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:86)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.common.assertion.AssertParameter.notNull(AssertParameter.groovy:26)
at io.restassured.internal.common.assertion.AssertParameter$notNull.callStatic(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:53)
at io.restassured.internal.common.assertion.AssertParameter$notNull.callStatic(Unknown Source)
at io.restassured.internal.RequestSpecificationImpl.header(RequestSpecificationImpl.groovy:869)
at com.api.auto.testcase.TC_API_CreateWork2.init(TC_API_CreateWork2.java:50)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- init -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC02_ValidateWorkId" time="0.000">
    <skipped/>
  </testcase> <!-- TC02_ValidateWorkId -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC02_ValidateMessage" time="0.000">
    <skipped/>
  </testcase> <!-- TC02_ValidateMessage -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC03_ValidateToken" time="0.000">
    <skipped/>
  </testcase> <!-- TC03_ValidateToken -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC03_ValidateNameOfWorkMatched" time="0.000">
    <skipped/>
  </testcase> <!-- TC03_ValidateNameOfWorkMatched -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC04_ValidateExperienceMatched" time="0.000">
    <skipped/>
  </testcase> <!-- TC04_ValidateExperienceMatched -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC04_ValidateUserType" time="0.000">
    <skipped/>
  </testcase> <!-- TC04_ValidateUserType -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC05_ValidateAccount" time="0.000">
    <skipped/>
  </testcase> <!-- TC05_ValidateAccount -->
  <system-out/>
  <testcase classname="com.api.auto.testcase.TC_API_CreateWork2" name="TC05_ValidateEducationMatched" time="0.000">
    <skipped/>
  </testcase> <!-- TC05_ValidateEducationMatched -->
  <system-out/>
</testsuite> <!-- com.api.auto.testcase.TC_API_CreateWork2 -->
